pkgs/development/libraries/yaml-cpp/default.nix

```diff
@@ -24,7 +25,7 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DYAML_CPP_BUILD_TOOLS=false"
-    "-DYAML_BUILD_SHARED_LIBS=${lib.boolToString (!stdenv.hostPlatform.isStatic)}"
+    "-DYAML_BUILD_SHARED_LIBS=${lib.boolToString (!static)}"
```

Comment:
`(lib.cmakeBool "YAML_BUILD_SHARED_LIBS" (!static))`

Reply:
Nice. I'll change. It makes the flags less beautiful though because the no longer line up. Unless you'd prefer I convert all of them to this form?

Reply:
Yeah, all those flags should use `(lib.cmakeBool )` IMO; but the others are literal strings so it's fine to leave that work for someone else

Reply:
I don't mind changing it myself if that's preferred.

Reply:
They'd get updated eventually. One would often abstain from rewriting things just for the sake of formatting so as not to inflate the diff. In this case there is only a total of 3 flags -> no excess review burden -> it's entirely your call:)

Reply:
I'll leave it as-is then.